docker/prod/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM golang:1-alpine3.9
+
+WORKDIR /go/src/github.com/unicsmcr/hs_auth
+
+COPY . .
+
+RUN apk add git
+RUN go get -d -v ./...
+RUN go install -v ./...
+RUN go build main.go
+
+ENV GOPATH /go
+
+EXPOSE 80
+
+ENTRYPOINT ./hs_auth
```

Comment:
Where does this script exist? Shouldn't it be running a Go command on main.go?

Reply:
It gets built by `go build main.go`

`go build main.go` compiles the app into a binary script. It's more efficient to run the binary than using the go cli to run the app

Reply:
Ok nvm, apparently running `go run main.go` is just as efficient. It's just a shortcut for compiling the code and running the executable.

I've replace the entrypoint with `go run main.go`